packages/notation/notation-tmpfiles.conf

```diff
@@ -0,0 +1,2 @@
+R /var/cache/notation - - - -
```

Comment:
Does this directory get automatically created?

Reply:
it gets created at runtime by notation: https://github.com/notaryproject/notation/blob/e49f07cd19d47fb5c79a0c56321c70bf13d525ef/internal/revocation/revocation.go#L34